src/activy/register.py

```
@@ -167,12 +175,156 @@ def registerStage9(device, templates):
+    """
+    Stage 12: Enter workplace.
+    """
+    search_bar = wait_for_element(device, className="android.widget.ImageView", instance=7, hint=True)
+    search_bar.click()
+    search_bar.set_text(workplace_name)
+
+    workplace_node = wait_for_element(device, contentDesc=workplace_name)
+    if workplace_node.exists:
+        workplace_node.click()
+        if not workplace_node.wait_gone(timeout=10):
+            raise TimeoutError(f"Confirm popup didnt appear.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)